src/main/java/com/mraof/minestuck/player/PlayerData.java

```diff
@@ -74,6 +74,7 @@ public static void onPlayerChangeDimension(PlayerEvent.PlayerChangedDimensionEve
 	private boolean givenModus;
 	private Modus modus;
 	private long boondollars;
+	private int caegerStorage;
```

Comment:
Ideally this could be handled by a capability or something else, instead of as a permanent stored component of every player(regardless of if they even touch the weapon). However I dont have the best advice or experience to know what changes to make at the moment

Reply:
the reason why it saves to the player is so that the player can switch what cain their using without having to build up their storage again. i also intend to have the caegers have other niche uses as an alternative form of currency as well as a tool for multiple other abilities i have planned so they're not going to be just used by this one weapon for long. i'll change it if i'm given a better way to get that effect.

Reply:
ultrakill marksman coin ultrakill marksman coin ultrakill marksman coin :B

Reply:
spoil something again i fucking dare you